components/list/list-item-mixin.js

```diff
@@ -126,9 +114,10 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			}
 
 			[slot="control-container"] {
+				pointer-events: none;
```

Comment:
`position: relative;` seems to bump this item out of control of `order`, so I left it on top but disallowed `pointer-events`, which seems to work fine.

Reply:
No objection to changing this, just calling this out if we run into any issues with it. I don't _think_ this `z-index` value was causing us any grief. It was just being used for rendering the separators in its `::before`/`::after` pseudo-elements. The element itself is not actually a container for item content, and so would not be setting a stacking context for anything else.

Reply:
The problem was that without the parent stacking context, `-1` puts it behind the list itself (or whatever it eventually hits) so the borders were no longer visible.